Makefile

```diff
@@ -157,4 +157,14 @@ clickhouse-monitor:
 clickhouse-monitor-plugin:
 	@mkdir -p $(BINDIR)
 	GOOS=linux $(GO) build -o $(BINDIR) $(GOFLAGS) -ldflags '$(LDFLAGS)' antrea.io/theia/plugins/clickhouse-monitor
-	
\ No newline at end of file
+
+.PHONY: theiactl
```

Comment:
I think we also need to modify githubactions to ensure the CLI is compiled and shipped with release assets

Reply:
Added

Reply:
thanks!